pages/utilities/number-base-changer.tsx

```diff
@@ -11,6 +11,7 @@ import CallToActionGrid from "@/components/CallToActionGrid";
 import Meta from "@/components/Meta";
 import { Combobox } from "@/components/ds/ComboboxComponent";
 import NumberBaseChangerSEO from "@/components/seo/NumberBaseChangerSEO";
+import { ArrowLeftRight } from "lucide-react";
 
 export default function Base64Encoder() {
```

Comment:
Please change this to NumberBaseChanger, this was totally on me, since I copy pasted component 😅
```suggestion
export default function NumberBaseChanger() {
```